### ISSUE TITLE
A screen that provides a back button would be great.

### ISSUE THREAD

**Issue by klaf1704** [TheFragebogen/TheFragebogen#5]
Here is some _not_ working demo code.
```
function ScreenUIElementsJump() {
    ScreenUIElements.apply(this, arguments);
    this.backCallback = null;
    this.sentBy = null;
    this.jumpIndex=1;
}
ScreenUIElementsJump.prototype = Object.create(ScreenUIElements.prototype);
ScreenUIElementsJump.prototype.constructor = ScreenUIElementsJump;

ScreenUIElementsJump.prototype.setOnBackCallback = function(backCallback){
    this.backCallback = backCallback;
};

ScreenUIElementsJump.prototype._sendOnBackCallback = function(){
    this.sentBy = -1;
    this.backCallback();
}

ScreenUIElementsJump.prototype.createUI = function () {
    this.node = document.createElement("div");
    this.node.className = this.className;

    for (var index in this.elements) {
        if (this.elements[index].createUI !== undefined) {
            var node = this.elements[index].createUI();
            if (node !== undefined && node !== null) this.node.appendChild(node);
        } else
            TheFragebogen.logger.warn(this.constructor.name + ".createUI():", "Element[" + index + "] has no 'createUI' method");
    }


    var button = document.createElement("input");
    button.type = "button";
    button.value = "Replay video";
    button.onclick = (this._sendOnBackCallback).bind(this);
    this.node.appendChild(button);

    this.node.appendChild(document.createElement("br"));

    var button = document.createElement("input");
    button.type = "button";
    button.value = "Next";
    button.onclick = (this._sendOnReadyCallback).bind(this);
    button.style.marginTop = "20px";
    this.node.appendChild(button);

    return this.node;
};
```

**Comment by dennisguse** [comment]
Pull request: https://github.com/TheFragebogen/TheFragebogen/pull/9

**Comment by dennisguse** [comment]
Merged #9.
